src/GafferImage/Display.cpp

```diff
@@ -268,7 +309,39 @@ class GafferDisplayDriver : public IECoreImage::DisplayDriver
 			Display::driverCreatedSignal()( driver.get(), parameters.get() );
 		}
 
-		ConstFloatVectorDataPtr getTile( const V2i &tileOrigin, size_t channelIndex )
+		struct TileCacheElement
+		{
+			TileCacheElement(): backBufferData( ImagePlug::blackTile()->copy() ), backBuffer( backBufferData->writable() ), dirty( false ), cachedTile( ImagePlug::blackTile() ), cachedForDataCount( 0 )
+			{
+			}
+
+			TileCacheElement( const TileCacheElement& other ) : TileCacheElement()
+			{
+				*this = other;
+			}
+
+			TileCacheElement& operator=( const TileCacheElement& other )
+			{
+				memcpy( &backBuffer[0], &other.backBuffer[0], backBuffer.size() * sizeof( float ) );
+				dirty = (bool)other.dirty;
+				cachedTile = other.cachedTile;
+				cachedForDataCount = other.cachedForDataCount;
+				return *this;
+			}
+
+			FloatVectorDataPtr backBufferData;
```

Comment:
Looks like we don't actually need `backBufferData` at all, since we only ever access `backBuffer` directly?